### PR TITLE
I18n IDs for PlayerLoader

### PR DESCRIPTION
**Pull request by jos3p**
As said in #3803 
> Please use ids for all i18n strings (older i18n code has omitted this, that will need to be updated separately).

I started with the strings from `interface/report/PlayerLoader` to see how it goes and manually copy/pasted the strings to their new id in all message.json files (or is there an easier way?) but willing to do the rest too over the next few days/weeks in one or more PRs.

### CODE REVIEW

File: src/interface/report/PlayerLoader.js [hunk 1]
```diff
@@ -211,16 +211,16 @@ class PlayerLoader extends React.PureComponent {
   }
 
   renderLoading() {
-    return <ActivityIndicator text={i18n._(t`Fetching player info...`)} />;
+    return <ActivityIndicator text={i18n._(t('interface.report.renderLoading.FetchingPlayerInfo')`Fetching player info...`)} />;
```
Comment (niseko) [inline src/interface/report/PlayerLoader.js]:
```suggestion
    return <ActivityIndicator text={i18n._(t('interface.report.renderLoading.fetchingPlayerInfo')`Fetching player info...`)} />;
```
very minor but I think everything should be in camelCase

File: src/interface/report/PlayerLoader.js [hunk 2]
```diff
@@ -325,7 +325,7 @@ class PlayerLoader extends React.PureComponent {
     return (
       <>
         {/* TODO: Refactor the DocumentTitle away */}
-        <DocumentTitle title={i18n._(t`${getFightName(report, fight)} by ${player.name} in ${report.title}`)} />
+        <DocumentTitle title={i18n._(t('interface.report.render.DocumentTitle')`${getFightName(report, fight)} by ${player.name} in ${report.title}`)} />
```
Comment (niseko) [inline src/interface/report/PlayerLoader.js]:
```suggestion
        <DocumentTitle title={i18n._(t('interface.report.render.documentTitle')`${getFightName(report, fight)} by ${player.name} in ${report.title}`)} />
```
same as above

**Comment by jos3p** [comment]
I have added the review changes

**Review by niseko — APPROVED** [review]
👍

**Pull request merged**